src/domains/chart/utils/transformDataAttributes.ts

```diff
@@ -553,3 +558,62 @@ export const getAttributes = (node: Element): Attributes => {
   const showValueOf = getAttributesDynamic(node)
   return { ...attributesStatic, showValueOf }
 }
+
+export const mapDefaultAggrMethod = (unit: string): string => {
+  if (unit.length === 0) {
+    return "sum"
+  }
+  const avgUnits: any = {
+    percentage: true,
+    percent: true,
+    "rotations/min": true,
+    ratio: true,
+    seconds: true,
+    "seconds ago": true,
+    milliseconds: true,
+    millisec: true,
+    ms: true,
+    "log2 s": true,
+    minutes: true,
+    hours: true,
+    interval: true,
+    ticks: true,
+    celsius: true,
+    c: true,
+    mhz: true,
+    hz: true,
+    volts: true,
+    kwh: true,
+    ampere: true,
+    amps: true,
+    dbm: true,
+    value: true,
+    stratum: true,
+    units: true,
+    watt: true,
+    temperature: true,
+    "random number": true,
+    rpm: true,
+    quadro: true,
+    "adv/item": true,
+    multiplier: true,
+    geforce: true,
+  }
+  if (avgUnits[unit.toLowerCase()]) {
```

Comment:
`avgUnits` could be also an array and then `avgUnits.includes(unit.toLowerCase())`, but that's very minor :)

Reply:
There is a small perf boost, since it avoids looping through the array - I suggest to leave it as is.

Reply:
I'm ok with this, I wasn't aware it's a tweak for performance. But IMO (in the future) it would be good if we don't do micro-optimisations like this often - it makes code less readable, gives low value, and if we have some performance problem, I suggest we fix them on a higher level (for example with useMemo). That's just my personal preference.

Btw. optimisation would be better if `avgUnits` were created outside the function :)

Reply:
https://github.com/netdata/dashboard/pull/127#discussion_r489446927